packages/mint-components/src/components/sqm-referred-registration/sqm-referred-registration.feature

```diff
@@ -0,0 +1,120 @@
+@author:truman
+Feature: Instant access referred registration
+
+    @motivating
+    Scenario: User's must provide an email to register
+        Given a user is trying to register as a referrer
+        When they do not provide an email
+        And they submit the form
+        Then an error appears telling the user to provide their email
+        When they provide an email
+        And they submit the form
+        Then they are logged in to the instant access widget
+
+    @motivating
+    Scenario: Users are notified if registration fails
+        Given a user is trying to register
+        And they have included their email
+        When an error occurs with the registration
+        Then an alert banner appears letting the user know about the error
+
+    @motivating
+    Scenario: Successful registration attributes referrals
```

Comment:
@lillyszatmary flagged this one in a couple cases but haven't heard anything back. is there a landmine here? What if the participant already exists in SaaSquatch, does the referral actually still get attributed?